blockchain/doc.go

```diff
@@ -70,7 +70,7 @@ violations through type assertions.  In addition, callers can programmatically
 determine the specific rule violation by examining the ErrorCode field of the
 type asserted blockchain.RuleError.
 
-Bitcoin Improvement Proposals
+Navcoin Improvement Proposals
 
```

Comment:
NavCoin Improvement Proposal doesn't lend itself to a very politically correct acronym 💦 

Pehraps NavCoin Enchanement Proposal would be better (NEP)